solution/backend/cmcs_regulations/templates/admin/login.html

```diff
@@ -19,9 +19,11 @@
         </style>
     {% endblock %}
    {{ block.super }}
-   <div class="okta_login_container">
-       Or Login With&nbsp; <a href="https://www.google.com/">EUA</a>
-   </div>
+   {% if EUA_FEATUREFLAG == 'true' %}
```

Comment:
when its changed to a boolean we can just do { % if EUA_FEATUREFLAG %}

Reply:
AWS parameter store only returns values as string. To simplify its just best to keep it consistent.